src/actions/ModelUpdateAction.php

```
@@ -0,0 +1,58 @@
+
+        return $pending;
+    }
+
+    private function persistHardwareSettings(array $pendingHardwareSettings): void
+    {
+        foreach ($pendingHardwareSettings as $hw) {
+            if (empty($hw['id'])) {
+                continue;
+            }
+            try {
+                HardwareSettings::perform('set-hardware-settings', [
```

PHPStan (2.1.54)

[error] Call to an undefined static method hipanel\modules\stock\models\HardwareSettings::perform().


(staticMethod.notFound)